app/domain/export/pdf/passes/paddle_pass/content.rb

```diff
@@ -85,6 +85,8 @@ def fetch_paddle_level_table_entries
 
     def club_name
       layer = group&.layer_group
+      return if layer.nil?
+
       layer.short_name || layer.name
```

Comment:
```suggestion
      layer&.short_name || layer&.name
```

Reply:
I respectfully disagree. Two safe-navigation operators are worse than one guard-statement. The guard communicates IMHO clearer that we intend to stop there if we do not have a layer.

Reply:
Fair enough :joy: :+1: